.github/workflows/jetbrains-integration-test.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: "1.19"
-      - uses: actions/setup-java@v2
+      - uses: actions/setup-java@v4
```

Comment:
v2 has some issues [[gha](https://github.com/gitpod-io/gitpod/actions/runs/14592745196)], upgrade to v4 helps, see [[gha](https://github.com/gitpod-io/gitpod/actions/runs/14601313860/job/40959939788)]